src/Options/index.js

```diff
@@ -25,6 +25,9 @@ export interface SchemaOptions {
   /* Is true if Parse Server will reject any attempts to modify the schema while the server is running.
   :DEFAULT: false */
   lockSchemas: ?boolean;
+  /* Keep indexes that are not defined in the schema and are present in the database. Set this to true if you are adding indexes manually so that it wont be dropped when you run schema migration
```

Comment:
```suggestion
  /* (Optional) Keep indexes that are not defined in the schema and are present in the database. Set this to `true` if you are adding indexes manually, so that they won't be removed when running schema migration. Default is `false`.
```

Could you please accept this change and run the definitions script to update the other files?